### DISCUSSION TITLE
Don't animate taiko notes on hit when the HUD is turned off

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#23090] [Ideas]
On osu!stable, when the HUD is turned off, the taiko notes don't get animated (the little throw animation), but on osu!lazer they are still being animated. I'd like to see them disappear just like on osu!stable when the HUD is turned off. Ideally, it could also be a toggle in the skin settings or something like that. I'm open for better suggestions.

**Comment by peppy** [comment]
I don't think this should be tied to HUD being on or off.

**Comment by reo-the-enby** [comment]
I don't think it should behave like that. In stable the animation is connected to the hp-bar which is why they don't get animated but since lazers animation have nothing with the hp-bar to do it should still be on.

**Reply by TheFearlessDeath500** [reply]
Well it could be optionally off or on depending on something in skin.ini perhaps?
